packages/account-v2/src/containers/TradingAssessmentForm/TradingAssessmentForm.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from 'react';
+import { Form, Formik } from 'formik';
+import { FormSubHeader } from '../../components/FormSubHeader';
+import { getTradingAssessmentQuestions } from '../../constants/tradingAssessmentQuestions';
+import { RiskToleranceWarningModal } from './RiskToleranceWarningModal';
+import { TestWarningModal } from './TestWarningModal';
+import { TradingAssessmentContainer } from './TradingAssessmentContainer';
+
+export const TradingAssessmentForm = () => {
+    const [shouldShowRiskTolerence, setshouldShowRiskTolerence] = useState(false);
+    const [shouldShowTestWarningModal, setShouldShowTestWarningModal] = useState(false);
+    const tradingAssessmentQuestions = getTradingAssessmentQuestions();
+
+    const handleAcceptRiskTolerence = () => {
+        setshouldShowRiskTolerence(false);
+    };
+
+    if (shouldShowRiskTolerence) {
+        return (
+            <RiskToleranceWarningModal handleSubmit={handleAcceptRiskTolerence} isModalOpen={shouldShowRiskTolerence} />
+        );
+    }
+
+    if (shouldShowTestWarningModal) {
+        return (
+            <TestWarningModal
+                handleSubmit={() => setShouldShowTestWarningModal(false)}
+                isModalOpen={shouldShowTestWarningModal}
+            />
+        );
+    }
+
+    return (
+        //TODO: implement onSubmit function and implement initialValues when tradingAssessment hooks is ready
+        // eslint-disable-next-line @typescript-eslint/no-empty-function
+        <Formik initialValues={{}} onSubmit={() => {}}>
+            <Form>
+                <div>
+                    <FormSubHeader>Trading assessment</FormSubHeader>
+                    {tradingAssessmentQuestions.map(item => {
+                        if (item.fieldType === 'radio') {
+                            return (
+                                <TradingAssessmentContainer
```

Comment:
Can `TradingAssessmentContainer` render a Radiobutton?

Reply:
this is basically why I asked about the usage of this form. we show radio in account creation flow, but not in account settings

Reply:
No it couldn't render a RadioButton, let me check whether we can make this container more dynamic 🤔

Reply:
I suppose we need two different containers and configs 🤔